Dockerfile/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM python:3.8-slim-buster
+
+RUN pip install --upgrade pip
+
+RUN apt-get update
+
+RUN apt-get install curl unzip --yes
+
+RUN apt-get update && apt-get install -y nodejs npm
```

Comment:
Add apt cache clean-up afterwards, it will shrink the image size.

Reply:
We can also combine these into a single RUN command to reduce the number of steps:
```
RUN apt-get update; \
    apt-get install curl unzip --yes \
    apt-get update && apt-get install -y nodejs npm
```

Reply:
@r0b2g1t  @picklelo  Thank you for your feedback! I have done most of the changes as suggested, but I'm facing the following problems:

- When I am trying to install nodejs via `RUN apt-get update && apt-get install -y curl nodejs npm unzip` , my script is hanging and docker stops working completely. To make it work again I have to restart my computer and check my Hyper-V settings. The same problem is occuring when I am trying to install Version 18 of Nodejs

- apt cache clean-up is not running and giving error while running the script

If possible, could you please look into it and direct me towards a possible solution?

Thank You!😁

Reply:
Hey @SaarthakMaini sure I will look to this issue and try it out and give you feedback. 🙂